src/nginx/status.h

```diff
@@ -41,6 +41,7 @@ namespace nginx {
 // The maximum number of esp objects.
 const int kMaxEspNum = 10;
 const int kMaxServiceNameSize = 256;
+const int kMaxServiceRolloutsInfoSize = 256;
```

Comment:
This looks like too small to me, do you have idea roughly how many config_ids will fit into this size?